Align with the current instruction for password length (#4436)
github/docs: content/github/authenticating-to-github/creating-a-strong-password.md

You must choose or generate a password for your {% data variables.product.product_name %} account that is at least:
- Eight characters long, if it includes a number and a lowercase letter, or
- 15 characters long with any combination of characters

To keep your account secure, we recommend you follow these best practices:

> - Use a password manager, such as [LastPass](https://lastpass.com/) or [1Password](https://1password.com/), to generate a password more than 15 characters.

Reviewer: ```suggestion
- Use a password manager, such as [LastPass](https://lastpass.com/) or [1Password](https://1password.com/), to generate a password of at least 15 characters.
```
